src/cli.ts

```diff
@@ -166,6 +167,12 @@ function makeCommand (process: NodeJS.Process): Command {
     ).default(
       Enums.ComponentType.Application
     )
+  ).addOption(
+    new Option(
+      '--workspaces <workspace...>',
```

Comment:
```suggestion
      '-w, --workspace <workspace-name...>',
```

Reply:
An option `'-ws, --workspaces'` is to be added, too. it should be piped to `npm-ls`, just like `'-w, --workspace ...'`